omnibus/config/software/librdkafka.rb

```diff
@@ -0,0 +1,16 @@
+name "librdkafka"
+default_version "2.0.2"
+
+version "2.0.2" do
+  source url: "https://github.com/confluentinc/librdkafka/archive/refs/tags/v2.0.2.tar.gz"
+  source sha256: "f321bcb1e015a34114c83cf1aa7b99ee260236aab096b85c003170c90a47ca9d"
+end
+
+build do
+  license "BSD-style"
+  license_file "https://raw.githubusercontent.com/confluentinc/librdkafka/master/LICENSE"
+
+  command "./configure --prefix=#{install_dir}/embedded --install-deps --source-deps-only"
+  command "make"
+  command "make install", :env => { "LD_RUN_PATH" => "#{install_dir}/embedded/lib" }
```

Comment:
I wasn't too sure about how these commands should be, I just used the same commands in the `librdkafka` instructions: https://github.com/confluentinc/librdkafka#building

Reply:
From what I understand [here](https://github.com/confluentinc/confluent-kafka-python/blob/master/INSTALL.md#install-from-source), we do not have to build `librdkafka` from the source, only `confluent-kafka` needs to be built from the source, we can install it using apt/brew/yum/vcpkg following this docs: https://github.com/confluentinc/confluent-kafka-python/blob/master/INSTALL.md#install-from-source